pkg/cloud-init/cloud-init.go

```diff
@@ -556,9 +556,9 @@ func GenerateLocalData(vmiName string, namespace string, data *CloudInitData) er
 	if err != nil {
 		return err
 	}
-	diskutils.RemoveFile(metaFile)
-	diskutils.RemoveFile(userFile)
-	diskutils.RemoveFile(networkFile)
+	diskutils.RemoveFilesIfExist(metaFile)
+	diskutils.RemoveFilesIfExist(userFile)
+	diskutils.RemoveFilesIfExist(networkFile)
 
```

Comment:
Why not using `diskutils.RemoveFilesIfExist(metaFile, userFile, networkFile)` ?

Reply:
look at the follow-up commit!